sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/UpdateCenterDeserializer.java

```diff
@@ -266,13 +263,20 @@ private void parsePlugins(Properties p, Sonar sonar, List<Plugin> plugins) {
       plugin.setSupportedBySonarSource(Boolean.valueOf(get(p, pluginKey, "supportedBySonarSource", false)));
       plugin.setBundled(Boolean.valueOf(get(p, pluginKey, "bundled", false)));
 
-      // do not add plugin without any version
-      if (!plugin.getAllReleases().isEmpty()) {
+      if (isPluginCompatibleWithAnySqRelease(plugin, sonar)) {
         plugins.add(plugin);
+      } else {
+        System.out.println("The plugin " + pluginKey + " is not compatible with any public SQ versions.");
```

Comment:
maybe we should use a Logger instead of system print ?

Reply:
totally! it's updated :)